src/platform/linux/wayland/window.rs

```diff
@@ -235,6 +235,11 @@ impl Window {
         }
     }
 
+
+    pub fn set_theme<T: Theme>(&mut self, theme: T) {
```

Comment:
Given there is locking inside, I don't think this method needs to take `&mut self`, `&self` should be enough.